lib/soapy_cake/admin_addedit.rb

```diff
@@ -173,6 +172,15 @@ def edit_campaign(opts)
 
     private
 
+    def addedit_offer_tier(add_edit_option, opts)
+      require_params(opts, %i(offer_id tier_id price_format_id offer_contract_id status_id))
+
+      opts = opts.merge(redirect_offer_contract_id: -1, add_edit_option: add_edit_option)
```

Comment:
Ideally, when I pass in an options hash into a method (or any object into any not obviously destructive method), I would not expect it to change. So, IMHO, the latter would be appropriate.